resources/app/src/components/people/DemographicsForm.vue

```diff
@@ -983,6 +983,9 @@ export default {
         //TODO revisit disadvanted logic
         disadvantaged: this.formdata.disadvantaged,
         disadvantaged_other: this.formdata.disadvantaged_other,
+        disadvantaged_opt_out: this.formdata.disadvantaged_opt_out,
```

Comment:
I went through the process of keeping the variables in the definition of `formdata` to be the same as the order they occur in the form itself (and in the Person.php) file... Exactly so things wouldn't get left out if we listed them again. But if you went back to destructuring assignment, that would also avoid the problem.